src/monorepo-workflow-operations.ts

```diff
@@ -64,6 +65,15 @@ export async function followMonorepoWorkflow({
   stdout: Pick<WriteStream, 'write'>;
   stderr: Pick<WriteStream, 'write'>;
 }) {
+  const { version: newReleaseVersion, firstRun } = await createReleaseBranch({
```

Comment:
What do you think about updating the tests for this file to make sure that these steps are performed?